_posts/2016/2016-05-13-geecon.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "GeeCON Kraków"
+image: blog-header/geecon.png
+date: "2016-05-13 07:16:34"
+categories: events
+authors: ["Mathias"]
+---
+
+Sometimes it is a good idea as a developer to step back and do something different for a few days. So you can broaden your view and reflect on your work. Visiting a conference is a good chance to do so - you can learn about new technology, listen to people who question things that you considered a given and see how other people solve similar problem.
```

Comment:
Visiting a conference is a good chance to do so. You can learn about new technology, listen to people who question things that you considered a given, and see how other people...